src/api/app/views/webui2/webui/attribute/_breadcrumb_items.html.haml

```diff
@@ -1,6 +1,6 @@
 = render partial: "webui/#{@container.model_name.singular}/breadcrumb_items"
 
-- if current_page?(index_attribs_path)
```

Comment:
We just want to know where we are. There is no need to generate a path. Especially if this can fail, e.g. when the object needed by the generated route does not exist. We had issues like [this](#6476) and alike a couple of times.

The only disadvantage of using `action_name` / `controller_name` that I see is that we have to adjust them manually whenever the routes change. But that doesn't happen often.